src/logic/currencyValues/store/middlewares/currencySelected.js

```diff
@@ -0,0 +1,31 @@
+// @flow
```

Comment:
The logic here is pretty straightforward (basically just calling `saveToStorage`), so this doesn't have to be a separate middleware

Reply:
done